runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -232,6 +231,43 @@ private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 					textLocation.getY());
 			}
 		}
+
+		if (plugin.isShowAgilityLevel())
+		{
+			if (!plugin.agilityMap.isEmpty() && plugin.agilityMap.keySet().contains(actor.getName()))
+			{
+				if (plugin.getAgilityFormat().equals(PlayerIndicatorsPlugin.AgilityFormats.ICONS))
+				{
+					int width = graphics.getFontMetrics().stringWidth(name);
+					int height = graphics.getFontMetrics().getHeight();
+					if (plugin.agilityMap.get(actor.getName()) >= 70)
+					{
+						OverlayUtil.renderImageLocation(graphics,
+								new Point(textLocation.getX() + 5 + width,
+									textLocation.getY() - height),
+								ImageUtil.resizeImage(agilityIcon, height, height));
+					}
+					if (plugin.agilityMap.get(actor.getName()) >= 84)
+					{
+						OverlayUtil.renderImageLocation(graphics,
+								new Point(textLocation.getX() + agilityIcon.getWidth() + width,
+									textLocation.getY() - height),
+								ImageUtil.resizeImage(agilityIcon, height, height));
+					}
+				}
+				else
+				{
+					name += " " + plugin.agilityMap.get(actor.getName());
+
+					int width = graphics.getFontMetrics().stringWidth(name);
+					int height = graphics.getFontMetrics().getHeight();
+					OverlayUtil.renderImageLocation(graphics, new Point(textLocation.getX() + 5 + width,
+																		textLocation.getY() - height),
```

Comment:
weird indentation

Reply:
How would you re-indent? textLocation is an argument to Point so it seems right to indent further here, plus my editor agrees. But I'll take a suggestion

Reply:
I'd do it like this, like it is just above https://i.imgur.com/3wm98Nv.png

Reply:
Better?